docs/examples/userguide/memoryviews/not_none.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+import typing
+
+def process_buffer(input_view: cython.int[:,:],
+                   output_view: typing.Optional[cython.int[:,:]] = None):
+
+    if output_view is None:
```

Comment:
You have slightly different levels of indentation between the .py and .pyx file - makes the text move a bit when switching between them.

Reply:
fixed.